src/modules/imageresizer/dll/ContextMenuHandler.h

```diff
@@ -0,0 +1,57 @@
+#pragma once
+
+#define ID_RESIZE_PICTURES 0
+#define RESIZE_PICTURES_VERBW L"resize"
+#include "stdafx.h"
+#include "resource.h"
+#include "ImageResizerExt_i.h"
+
+#if defined(_WIN32_WCE) && !defined(_CE_DCOM) && !defined(_CE_ALLOW_SINGLE_THREADED_OBJECTS_IN_MTA)
+#error "Single-threaded COM objects are not properly supported on Windows CE platform, such as the Windows Mobile platforms that do not include full DCOM support. Define _CE_ALLOW_SINGLE_THREADED_OBJECTS_IN_MTA to force ATL to support creating single-thread COM object's and allow use of it's single-threaded COM object implementations. The threading model in your rgs file was set to 'Free' as that is the only threading model supported in non DCOM Windows CE platforms."
+#endif
+
+using namespace ATL;
```

Comment:
@ryanbodrug-microsoft is this a 0.16 blocker? If so, can this be done in a separate PR since it looks like the ContextMenuHandler files would need some refactoring?

Reply:
@arjunbalgovind / @udit3333 .  I don't see this as a .16 blocker.  My preference would to be to get the code in as is and make the ATL-> COM change as part of either #1217 or #1051.   As we didn't write this code, I'd like to get it in as is so we have a baseline to compare against if we accidentally introduce a regression doing these chages. 

@yuyoyuppe:  Is there any specific concern you have with ATL here, or do you just see this as a step towards creating a common base layer?

Reply:
yes, it'd be just a step towards creating a common base layer, I don't mind merging it as is of course.